CMakeLists.txt

```diff
@@ -66,14 +69,8 @@ if(NOT WIN32)
 
 endif()
 
-find_program(NASM_BIN nasm)
```

Comment:
There is another check for NASM in bfsdk/cmake/CMakeGlobal_Project.txt. Looks like I left it commented out on accident, I will put it back in for now. In a future PR, I would like to move the checks for NASM to an x86 specific toolchain file, as it would not make sense to require nasm if only building for ARM.

Reply:
Huh.... yeah your right. Originally the super-build didn’t include the project file so that nasm check was not there.